assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -636,5 +636,13 @@ export const MESSAGES = {
         </ul>
       ),
     },
+    VIDEO_MISSING_POSTER: {
```

Comment:
As per https://github.com/google/web-stories-wp/issues/7571#issuecomment-843486280 there is already `MISSING_VIDEO_POSTER` some lines above...

Reply:
that's for the whole story, this is specific to videos in the story

Reply:
Do you mean `MISSING_POSTER`? What I am referring to is `MISSING_VIDEO_POSTER`.

https://github.com/google/web-stories-wp/blob/c646e5bd76d3825d7ef355d1720f4d4ad741b6d9/assets/src/edit-story/app/prepublish/constants.js#L194-L207

That's not currently used anywhere actually. As per https://github.com/google/web-stories-wp/issues/7571#issuecomment-843486280 it seems like it was never properly implemented.

Reply:
fixed, removed duplicate. kept the part i added since it has current copy and is part of the media group